src/picotorrent/ui/torrentdetailsoverviewpanel.cpp

```diff
@@ -84,7 +85,7 @@ class CopyableStaticText : public wxStaticText
     }
 };
 
-TorrentDetailsOverviewPanel::TorrentDetailsOverviewPanel(wxWindow* parent, wxWindowID id, int cols, bool showPieceProgress)
+TorrentDetailsOverviewPanel::TorrentDetailsOverviewPanel(wxWindow* parent, wxWindowID id, std::shared_ptr<pt::Core::Configuration> cfg, int cols, bool showPieceProgress)
```

Comment:
```suggestion
TorrentDetailsOverviewPanel::TorrentDetailsOverviewPanel(wxWindow* parent, wxWindowID id, bool isDarkMode, int cols, bool showPieceProgress)
```
Passing a bool for dark mode makes it a bit cleaner and we don't have to store a reference to the whole config instance.